coda/coda_validate/views.py

```diff
@@ -172,7 +174,8 @@ def stats(request):
     # make a set of data that makes sense for the heatmap
     all_v = Validate.objects.all().order_by('last_verified')
     sums_by_date = {}
-    first_day, last_day = all_v[0], all_v.reverse()[0]
+    last_day = all_v[0], all_v.reverse()[0]
+    last_day = last_day[-1]
```

Comment:
These two lines seem strange. Should they be replaced with something like `last_day = all_v[-1]`?